pkgs/by-name/on/onscripter-en/package.nix

```diff
@@ -1,66 +1,88 @@
 {
   lib,
-  SDL,
-  SDL_image,
-  SDL_mixer,
-  SDL_ttf,
+  buildEnv,
   fetchFromGitHub,
-  freetype,
-  libjpeg,
-  libogg,
-  libpng,
-  libvorbis,
+  makeBinaryWrapper,
+  onscripter,
   pkg-config,
-  smpeg,
-  stdenv,
+  xcbuildHook,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
-  pname = "onscripter-en";
-  version = "20110930";
+let
+  inherit (onscripter) stdenv;
+in
+onscripter.overrideAttrs (
```

Comment:
We should [avoid introducing new uses of `overrideAttrs`](https://github.com/NixOS/nixpkgs/blob/df65e0ab4ab644b9a9606e8250f6f26957ff3919/pkgs/README.md#overrideattrs-and-overridepythonattrs). This would ideally be a flag in the `onscripter` package, but considering this is a version from 2011, has no maintainer, and the upstream source is apparently gone, perhaps it would be better to simply drop this package?

It seems like there are active forks, though:

* https://github.com/Galladite27/ONScripter-EN
* https://github.com/insani-org/onscripter-insani

Reply:
I've now removed ONScripter EN because 1. there should be English support in ONScripter upstream, 2. I'm not a user of ONScripter EN so it's better for actual users to test and maintain the package if needed.